spokestack/nlu/tflite.py

```diff
@@ -59,36 +63,31 @@ def __call__(self, utterance: str) -> Dict[str, Any]:
         # retrieve slots from the tagged postions and decode slots back
         # into original values
         slots = [
-            [token_id, tag[2:]] for token_id, tag in zip(input_ids, tags) if tag != "o"
+            (token_id, tag[2:]) for token_id, tag in zip(input_ids, tags) if tag != "o"
         ]
         slot_map: dict = {}
-        for token, tag in slots:
+        for (token, tag) in slots:
+            print(tag)
```

Comment:
Debug code

Reply:
And this block could also be simplified to
```
if tag in slot_map:
    slot_map[tag].append(token)
else:
    slot_map[tag] = [token]
```
Not a huge savings; just pointing it out.